### PR TITLE
i18n(es): Translate `showcase.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description

- What does this PR change? Give us a brief description.
Translate `showcase.mdx`

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/resources/showcase.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Mostrador de Starlight
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/resources/showcase.mdx:2]:
```suggestion
title: Exhibición de Starlight
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Thanks for the PR @Waxer59 💜, left a few one suggestion. Lets Get This Merged!

**Comment by Waxer59** [comment]
Cambios aplicados, gracias por la revisión 🤠!

**Pull request merged**